src/libraries/System.Net.Http/tests/StressTests/HttpStress/Program.cs

```diff
@@ -158,6 +162,9 @@ private static async Task<ExitCode> Run(Configuration config)
 
             string GetAssemblyInfo(Assembly assembly) => $"{assembly.Location}, modified {new FileInfo(assembly.Location).LastWriteTime}";
 
+            Type msQuicApiType = typeof(QuicConnection).Assembly.GetType("System.Net.Quic.MsQuicApi");
+            string msQuicLibraryVersion = (string)msQuicApiType.GetProperty("MsQuicLibraryVersion", BindingFlags.NonPublic | BindingFlags.Static).GetGetMethod(true).Invoke(null, Array.Empty<object?>());
```

Comment:
@ManickaP unfortunately, this line fails with `NullReferenceException` in release builds. I assume the `MsQuicLibraryVersion` is being trimmed away, since it's unused in the production code.

We could workaround this by moving the version extraction to a shared test-utility (together with the MaxWorkerQueue hack helper), but IMHO it's not worth it unless we think that it's super-important to know the msquic version for stress test troubleshooting.

For now I'm just removing it.

Reply:
is it possible that there is image without MsQuic?

Reply:
What do you mean? If you are suspecting lack of msquic as the root cause of this error - it's not the case. Removing code messing with MsQuicLibraryVersion makes things work (= HTTP/3 stress runs).

Reply:
Another option would be to tell the trimmer to keep it there via `DynamicDependency`. And this is just one string.

Reply:
We should try to clarify whether test-only usage of `DynamicDependency` is ok, without that, I prefer to defensively assume it's not.